charts/csi-cloudscale/templates/storageclass.yaml

```diff
@@ -0,0 +1,24 @@
+{{- $csi := .Values.csi }}
+{{- $provisioner := .Values.provisioner }}
+{{- range $storageClass := .Values.csi.storageClasses }}
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: {{ $storageClass.name }}
+{{- if $storageClass.default }}
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+{{- end }}
+provisioner: {{ $storageClass.provisioner | default $provisioner.name }}
+allowVolumeExpansion: {{ $storageClass.allowVolumeExpansion | default $csi.allowVolumeExpansion }}
+parameters:
+  csi.cloudscale.ch/volume-type: {{ $storageClass.volumeType }}
```

Comment:
It's required to have a fstype defined. Otherwise at least on OpenShift the volume is not accessible after the creation, because the fileystem permissions are not going to be initialized. See projectsyn/component-csi-cloudscale#14. This is given since the `csi-provisioner` get the default of èxt4`.

We are not using anything different than `fsType: ext4` right now. However there might be a reason to override this. So having it configurable could make sense.

Reply:
To support filesystems other than ext4, some changes to the driver itself would be required, therefore I prefer to have this not configurable through the helm chart. The issues regarding OpenShift should have been addressed by #26 and #27 respectively.  Let me know if you don't agree.

Reply:
Fine by me.